src/node_javascript.cc

```diff
@@ -6,29 +6,53 @@
 
 namespace node {
 
+using v8::Isolate;
 using v8::Local;
+using v8::MaybeLocal;
 using v8::NewStringType;
 using v8::Object;
 using v8::String;
 
+template <typename T, size_t N, T P>
```

Comment:
Tiny suggestion: Could we be a bit more explicit and rename `P` to `Ptr` or `Pointer`? :smile: